cmake/bob.cmake

```diff
@@ -174,7 +174,7 @@ endmacro(bob_end_subdir)
 function(bob_end_package)
   include(CMakePackageConfigHelpers)
   set(INCLUDE_INSTALL_DIR include)
```

Comment:
I do wonder if this `include` should be replaced with CMAKE_INSTALL_INCLUDEDIR or CMAKE_INSTALL_OLDINCLUDEDIR? Not sure if that's overkill since honestly that path likely won't change.

https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

Reply:
I think using `CMAKE_INSTALL_INCLUDEDIR` here is the clean/consistent approach.  Thank you.